lib/posix-process/signal/alarm.c

```diff
@@ -0,0 +1,14 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2023, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+
+#include <uk/essentials.h>
+#include <uk/syscall.h>
+
+UK_SYSCALL_R_DEFINE(unsigned int, alarm,
+		    unsigned int __unused, seconds)
```

Comment:
Ditto about `__unused`
And since it's a stub maybe we should add a warning like you did in other places?

Reply:
Yes, I'll add a  `UK_WARN_STUBBED()`.